packages/videojs.dev/package.json

```diff
@@ -1,17 +1,51 @@
 {
   "name": "videojs.dev",
-  "private": true,
   "version": "0.0.0",
-  "type": "module",
+  "private": true,
   "scripts": {
-    "build": "bun run build:source",
-    "build:source": "bun run bun.build.ts",
+    "docusaurus": "docusaurus",
+    "start": "docusaurus start",
+    "build": "docusaurus build",
```

Comment:
My main uncertainty is how we want to incorporate NX into this package.json. Docusaurus has its own build command, so I'd love feedback on how we think it is best to wrap it all together.